content/en/getting_started/integrations/google_cloud.md

```diff
@@ -351,3 +353,4 @@ After setting up CSM, toggle the **Enable Resource Collection** option under the
 [62]: https://app.datadoghq.com/event/overview
 [63]: https://cloud.google.com/service-usage/docs/access-control#serviceusage.serviceUsageConsumer
 [64]: https://github.com/GoogleCloudPlatform/terraform-gcp-datadog-integration
+[65]: /integrations/google_cloud_platform/#expanded-bigquery-monitoring
```

Comment:
I presume the contents for this section will be added later?

Reply:
Yes, that's correct, I'm awaiting a separate PR to be merged first